pp/go/relabeler/head/catalog/catalog.go

```diff
@@ -162,6 +187,7 @@ func (c *Catalog) SetCorrupted(id string) (_ *Record, err error) {
 	}
 
 	if r.corrupted {
+		c.corruptedHead.Inc()
```

Comment:
This corruption is already logged on changing this field. Don't increment counter if there is no changes made